### PR TITLE
526/time remaining in batch

### LINKED ISSUE #526: Display time remaining in batch
There was a task for that before https://github.com/gnosis/dex-react/issues/235

We decided to add it and use local time instead of blockchain time.

### Notes:

Also in RF.1.2 Timer of the current batch

using local time of the browser

### Estimation:
*1.5*

### PR DESCRIPTION
**Pull request by alfetopito**
Closes #526 

- Uses local time to calculate time remaining in batch
- Code on `feature/ui-redesign` branch since it has the time already on the UI
- NOT passing tests/build due to linter issues. Will be addressed on https://github.com/gnosis/dex-react/issues/534
- Able to run locally with `yarn mock` and see the clock ticking :)

### Note:
Not sure there's a smarter way to format seconds into XXm XXs from seconds rather than from a Date or timestamp. Open to suggestions.

### CODE REVIEW

File: src/utils/time.ts [hunk 1]
```diff
@@ -35,6 +35,32 @@ export function formatDateFromBatchId(batchId: number): string {
...
 
+/**
+ * Calculates seconds remaining in current batch
+ * Assumes local time is accurate and can be used as source of truth
+ */
+export function getSecondsRemainingInBatch(): number {
+  return BATCH_TIME - (getEpoch() % BATCH_TIME)
+}
+
+export function formatSecondsToMinutes(seconds: number): string {
```
Comment (anxolin) [inline src/utils/time.ts]: I would call this just `formatSeconds` since it can be missleasing to think that `30` --> `30s` and not `0.5m`

Comment (Velenir) [inline src/utils/time.ts]: Cool but wouldn't `date-fns` have a function for that

Comment (alfetopito) [reply src/utils/time.ts]: That's what I though, but as far as I can see, their functions are Date/timestamp based. I would need to add those seconds to now() and compare against it with something like https://date-fns.org/v2.9.0/docs/formatDistanceToNow

Or is there a simpler way?

Comment (alfetopito) [reply src/utils/time.ts]: See https://github.com/gnosis/dex-react/pull/542

Comment (Velenir) [reply src/utils/time.ts]:
```js
const date = new Date(seconds * 1000)
const minutes = date.getMinutes()
const seconds = date.getSeconds()
```
Is it easier? Not really

As for `date-fn` the best I found
```sh
> d.formatDistance(17000, 0, { includeSeconds: true})
'less than 20 seconds'
> d.formatDistance(97000, 0, { includeSeconds: true})
'2 minutes'
> d.formatDistance(77000, 0, { includeSeconds: true})
'1 minute'
```

which is not ideal

Comment (alfetopito) [reply src/utils/time.ts]: Not much better in my opinion. I'd like to keep the current version in this case.

**Pull request merged**